lib/ui.rb

```diff
@@ -37,7 +37,7 @@ def goodbye_message
   end
 
   def win_message(final_player)
-    "#{final_player} is the winner!"
+    "#{final_player.marker} is the winner!"
```

Comment:
**praise:** Nice work changing the player into an object!

Reply:
Thank you!